lib/pages/timetable/widget/timetable_tabbar.dart

```diff
@@ -11,38 +11,42 @@ class TimeTableTabBar extends StatefulWidget {
 }
 
 class _TimeTableTabBarState extends State<TimeTableTabBar> {
- @override
-Widget build(BuildContext context) {
-  return Padding(
-    padding: const EdgeInsets.symmetric(horizontal: 20.0), 
-    child: TabBar(
-      controller: widget.controller,
-      unselectedLabelStyle: const TextStyle(color: Colors.grey),
-      labelStyle: const TextStyle(color: Colors.white, fontWeight: FontWeight.bold),
-      indicatorColor: Colors.transparent,
-      labelPadding: EdgeInsets.zero,
-      indicator: BoxDecoration(
-        color: AppColors.primaryColor,
-        borderRadius: BorderRadius.circular(18.0),
-      ),
-      dividerColor: Colors.transparent,
-      indicatorSize: TabBarIndicatorSize.tab,
-      tabs: List.generate(6, (index) {
-        bool isSelected = widget.controller.index == index;
-        return Tab(
-          child: AnimatedContainer(
-            duration: const Duration(milliseconds: 300),
-            width: isSelected ? 50.0 : 30.0, // Increase width when selected
-            height: isSelected ? 50.0 : 40.0, // Increase height when selected
-            alignment: Alignment.center,
-            child: Text(
-              ["MON", "TUE", "WED", "THU", "FRI", "SAT"][index],
-            ),
-          ),
-        );
-      }),
-    ),
-  );
-}
+  @override
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
 
+    final double responsiveTabWidth = screenWidth / 7;
+
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 20.0),
+      child: TabBar(
+        controller: widget.controller,
+        unselectedLabelStyle: const TextStyle(color: Colors.grey),
+        labelStyle:
+            const TextStyle(color: Colors.white, fontWeight: FontWeight.bold),
+        indicatorColor: Colors.transparent,
+        labelPadding: EdgeInsets.zero,
+        indicator: BoxDecoration(
+          color: AppColors.primaryColor,
+          borderRadius: BorderRadius.circular(18.0),
+        ),
+        dividerColor: Colors.transparent,
+        indicatorSize: TabBarIndicatorSize.tab,
+        tabs: List.generate(6, (index) {
+          bool isSelected = widget.controller.index == index;
+          return Tab(
+            child: AnimatedContainer(
+              duration: const Duration(milliseconds: 300),
+              width: isSelected ? responsiveTabWidth + 10 : responsiveTabWidth,
+              height: isSelected ? 50.0 : 40.0,
+              alignment: Alignment.center,
+              child: Text(
+                ["MON", "TUE", "WED", "THU", "FRI", "SAT"][index],
```

Comment:
check if animatedContainer is still needed here

Reply:
Yes!